Tracing http client ip documentation update (#15686)
DataDog/documentation: content/en/security_platform/application_security/setup_and_configure.md

{{< /programming-lang-wrapper >}}

## Configuring a client IP header

> ASM automatically attempts to resolve `http.client_ip` from a number of well known headers, such as `X-Forwarded-For`. If you use a custom header for this field, or want to bypass the resolution algorithm, set the `DD_TRACE_CLIENT_IP_HEADER` environment variable and the library looks only in the specified header for the client IP.

Reviewer: ```suggestion
ASM automatically attempts to resolve `http.client_ip` from several well-known headers, such as `X-Forwarded-For`. If you use a custom header for this field, or want to bypass the resolution algorithm, set the `DD_TRACE_CLIENT_IP_HEADER` environment variable and the library only checks the specified header for the client IP.
```
